packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -816,10 +828,12 @@ export class ReadAlongComponent {
    * To add a new term, add a new key to the translations object. Then add 'eng' and 'fr' keys
    * and give the translations as values.
    *
-   * @param word
-   * @param lang
+   * @param word short name for the text to fetch
+   * @param lang language code
+   * @param path (optional) the path/file/href the error message applies to
+   * @param assetType (optional) type of assert the error message applies to
    */
-  returnTranslation(word: string, lang?: InterfaceLanguage): string {
+  returnTranslation(word: string, lang?: InterfaceLanguage, path?: string, assetType?: string): string {
```

Comment:
if we are going to support more than 2 languages this way of doing translations will get ugly fast, but for the moment it will be okay...

Reply:
Yes, I want to run prettier, but it changes about 90% of this file, so not before we've merged in the last feature changes. Anything outstanding will not be rebasable after we run prettier.

Reply:
But I will run prettier on just the new lines, that's not going to conflict with anything in the future.